Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -56,6 +56,13 @@ open class NavigationMapView: UIView {
             }
         }
     }
+
+    /**
+     The tolerance value used for configuring the underlying map source of route line, maneuver arrow and restricted areas.
+
+     Higher means simpler geometries and faster performance.
+     */
+    public var tolerance: Double = RouteSourcesTolerance
```

Comment:
Should we mention here that it only affects new objects? Or maybe we could do the `didSet` method to update existing sources?

Reply:
It turns out that the `tolerance` property is read-only, so we only can manipulate it before adding a source to the map. It makes sense to mention this in the docs.